cli/README.md

```diff
@@ -37,7 +37,7 @@ Note that Oxigraph CLI was previously named Oxigraph Server before version 0.4.
 
 ## Installation
 
-You need to have [a recent stable version of Rust and Cargo installed](https://www.rust-lang.org/tools/install).
+You need to have [a recent stable version of Rust and Cargo installed](https://www.rust-lang.org/tools/install) as well as Clang and LLVM for the RocksDB Rust bindings.
```

Comment:
```suggestion
You need to have [a recent stable version of Rust and Cargo installed](https://www.rust-lang.org/tools/install) as well as Clang for the RocksDB Rust bindings.
```

Reply:
@Tpt you are right! Contrary to what is [written in the RocksDB README](https://github.com/rust-rocksdb/rust-rocksdb?tab=readme-ov-file#requirements), Clang alone is enough.